app/assets/javascripts/pages/users-getting-started.js

```diff
@@ -49,36 +49,13 @@ Diaspora.Pages.UsersGettingStarted = function() {
       return confirmation;
     });
 
-    /* ------ */
-    var autocompleteInput = $("#follow_tags");
     var tagFollowings = new app.collections.TagFollowings();
-
-    autocompleteInput.autoSuggest("/tags", {
-      selectedItemProp: "name",
-      selectedValuesProp: "name",
-      searchObjProps: "name",
-      asHtmlID: "tags",
-      neverSubmit: true,
-      retrieveLimit: 10,
-      selectionLimit: false,
-      minChars: 2,
-      keyDelay: 200,
-      startText: "",
-      emptyText: "no_results",
+    new Diaspora.TagsAutocomplete("#follow_tags", {
       selectionAdded: function(elem){tagFollowings.create({"name":$(elem[0]).text().substring(2)})},
       selectionRemoved: function(elem){
         tagFollowings.where({"name":$(elem[0]).text().substring(2)})[0].destroy();
         elem.remove();
       }
```

Comment:
Add `preFill: gon.preloads.tagsArray` and add the gon preload to the getting_started method of the users controller.